interface/patient_file/summary/dashboard_header.php

```diff
@@ -20,15 +20,11 @@
 use OpenEMR\Common\Csrf\CsrfUtils;
 use OpenEMR\Common\Twig\TwigContainer;
 
-$url_webroot = $GLOBALS['webroot'];
-$portal_login_href = $GLOBALS['webroot'] . "/interface/patient_file/summary/create_portallogin.php";
```

Comment:
good call on removing this :)

![2DV](https://user-images.githubusercontent.com/278968/172433357-1621bcda-bba3-4e56-92c5-e67e26a32862.gif)

Reply:
pretty sure the delete html partial would have to have these set in view args to be in scope

thanks @bradymiller & @sjpadgett